src/swap/components/SwapProvider.tsx

```diff
@@ -125,6 +125,11 @@ export function SwapProvider({
     }
   }, [hasHandledSuccess, lifeCycleStatus.statusName]);
 
+  let maxSlippage: string;
+  if (lifeCycleStatus.statusData && 'maxSlippage' in lifeCycleStatus.statusData) {
```

Comment:
I am a bit confused what's going on here.

Reply:
This is all to avoid Typescript errors. For example, if I just have:

` const maxSlippage = String(lifeCycleStatus.statusData.maxSlippage)`

Then I'll see the following errors: 

```
> -  'lifeCycleStatus.statusData' is possibly 'null'.ts(18047)
> - 'lifeCycleStatus.statusData' is possibly 'null'.ts(18047)
> - Property 'maxSlippage' does not exist on type 'SwapError | { isMissingRequiredField: boolean; } | { amountFrom: string; amountTo: string; tokenFrom?: Token | undefined; tokenTo?: Token | undefined; isMissingRequiredField: boolean; } | { ...; } | { ...; } | { ...; }'.Property 'maxSlippage' does not exist on type 'SwapError'.ts(2339)
```